documentation/configuration/in-process-features-configuration.md

```diff
@@ -108,9 +109,81 @@ Similarly, the call stacks feature can be individually disabled by setting the s
   <summary>Kubernetes Environment Variables</summary>
   
   ```yaml
-  - name: InProcessFeatures__Enabled
+  - name: DotnetMonitor_InProcessFeatures__Enabled
     value: "true"
   - name: DotnetMonitor_InProcessFeatures__CallStacks__Enabled
     value: "false"
   ```
 </details>
+
+## Exceptions History
+
+The exceptions history feature is individually enabled by setting the `Enabled` property of the `Exceptions` section to `true`:
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "InProcessFeatures": {
+      "Exceptions": {
+        "Enabled": true
+      }
+    }
+  }
+  ```
+</details>
+
+<details>
+  <summary>Kubernetes ConfigMap</summary>
+  
+  ```yaml
+  InProcessFeatures__Exceptions__Enabled: "true"
+  ```
+</details>
+
+<details>
+  <summary>Kubernetes Environment Variables</summary>
+  
+  ```yaml
+  - name: DotnetMonitor_InProcessFeatures__Exceptions__Enabled
+    value: "true"
+  ```
+</details>
+
+Similarly, the exceptions history feature can be individually disabled by setting the same property to `false`:
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "InProcessFeatures": {
+      "Enabled": true,
```

Comment:
Is it worth calling out what happens if you explicitly set `InProcessFeatures:Enabled = false`? If we add this it also might be better to live a bit higher up in the doc in the `InProcessFeatures` section

Reply:
I'll add that in the next PR